src/cli/main.c

```diff
@@ -119,6 +119,8 @@ static void icc_types()
   fprintf(stderr, " HLG_REC2020\n");
   fprintf(stderr, " PQ_P3\n");
   fprintf(stderr, " HLG_P3\n");
+  fprintf(stderr, " ACES\n");
```

Comment:
ACES is an organization, not a color space.

Reply:
I admit a took a shortcut here - "ACES" was the "one" color space [before 2016](http://j.mp/TB-2014-012), so I kinda kept that anachronism here to avoid verbosity. And it's not an organization, it is "Academy Color Encoding Specification".

![image](https://user-images.githubusercontent.com/4973094/183359077-2d4f2ce7-042a-4654-8cad-1c64e62774a3.png)

Shall we go changing "PQ_P3" to "SMPTE-ST-2084_DCI-P3-D65" as well?

Reply:
ACES is not a specific name. PQ_P3 is : we have the transfer function and the primaries explicitly mentioned.

Reply:
It is the former/informal name for ACES2065-1 as quoted. That one has a transfer function (none, i.e. linear) and primaries (AP0) set in stone, no need to be more explicit.

I can certainly change this to make you happy, but let's first establish if there is any utility/benefit for dt in this change and do the bike-shedding after please.